tests/org.eclipse.n4js.xpect.tests/xt-tests/scoping/TypeVsValueScopeImport.n4js.xt

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) 2021 NumberFour AG.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   NumberFour AG - Initial API and implementation
+ */
+
+/* XPECT_SETUP org.eclipse.n4js.xpect.tests.N4jsXtTest
+   
+
+	Workspace {
+		JavaProject "N4JSXpectTests" {
+			Folder "model" {
+				ThisFile {}
+				File "TypeVsValueScopeExport.n4jsd" {}
+			}
+			File "package.json" { from="../package.json" }
+		}
+	}
+   
+   
+   END_SETUP
+ */
+
+
+import {I1, T1} from "TypeVsValueScopeExport"
+
+
+// XPECT noerrors -->
+interface I1_TYPE extends I1 {}
+// XPECT noerrors -->
+let i1_VAL = I1;
+
+
+// XPECT noerrors -->
+let t1_TYPE : T1 = null;
+// XPECT noerrors -->
+let t1_VAL = T1;
```

Comment:
In the 4 above test cases, maybe it would be good to ensure that we actually link to the correct element. Could this be done with Xpect method "hyperlink" or "link"?